mycroft/stt/__init__.py

```diff
@@ -547,6 +548,10 @@ def get_response(self, response):
         return response.json()["result"]["hypotheses"][0]["transcript"]
 
 
+def load_stt_plugin(module_name):
```

Comment:
why a helper function here?  couldn't this one line of code be executed on line 581?

Reply:
Basically for keeping the `create()` method below clear and readable